tests/coprint/input.json

```diff
@@ -0,0 +1,20 @@
+{
+    "items": [{
+            "filename": "./tests/coprint/Christmas.stl",
+            "count": 1,
+            "coprint": {
+                "filename": "./tests/coprint/Merry.stl",
+                "transformation": [
+                    [1, 0, 0, -60],
+                    [0, 1, 0, -343],
+                    [0, 0, 1, 0],
+                    [0, 0, 0, 1]
+                ]
+            }
```

Comment:
```suggestion
            "co_print": [
                {
                    "filename": "./tests/coprint/Merry.stl",
                    "transformation": [
                        [1, 0, 0, -60],
                        [0, 1, 0, -343],
                        [0, 0, 1, 0],
                        [0, 0, 0, 1]
                    ]
                }
            ]
```

Reply:
More than one object can be co-printed. The `Merry.stl` is just the first and only one in the list in this example.